content/blog/intro-to-gamedev/index.md

```diff
@@ -0,0 +1,560 @@
+
+Any object (text, lines, sprites) can be displayed this way. Since our use is pretty limited, there's not really much more to say about this system, but it has a lot more useful functions which I recommend exploring.
+
+### Note about coordinates
+
+PixiJS uses a coordinate system that comprises the 2 usual suspects: The X coordinate, and the Y coordinate. However, it is important to keep in mind that the (0,0) coordinate is on the **top left**, and **the X coordinate increases to the right**, whereas **the Y component increases as we go down on the screen**. This is the most common coordinate system in game development tools so we'll leave it as it is, but as you can imagine there are things you can do to translate and modify the system as desired.
+
+## Main gameplay loop
+
+We've seen what the gameplay loop looks like in general terms, but how do we implement the main update function (ie: `UpdateWorld()`)? The answer is pretty simple, the `PIXI.Application` object that we created contains a field called `ticker` of type [`PIXI.Ticker`](https://pixijs.download/dev/docs/PIXI.Ticker.html), which lets us register different tickers that will run on each animation frame (for example, 60 times per second). After its execution, the frame is rendered (that is,the objects we set on the `stage`).
+
+This rendering process can be done at hand by [getting the renderer object and calling `render()`](https://pixijs.download/dev/docs/PIXI.Ticker.html#.shared), but we will avoid this and let PixiJS do it automatically after running the ticker. This way, we are essentially coupling the `UpdateWorld()` and `RenderWorld()` of our abstract game loop, which will work just fine for our needs.
+
+Let's create a set of static methods and fields where we will group our game logic in our `GameApp` class. We know we want to know if the player is in a game over state, and what score they are achieving, so we'll define fields for those values.
+
+Regarding methods, for now we'll have one method for setup and one for the world updates, which we will implement through the PixiJS ticker:
+
+```ts
+export class GameApp {
+    static GameOver: boolean = false;
+    static PressedSpace: boolean = false; // whether the player pressed space
+    static Score: number = 0;
+
+    constructor(parent: HTMLElement, width: number, height: number) {
+        // code we added in the previous section goes here
+
+        // register the event for key presses (PixiJS does not handle input, so we do it through the browser)
+        window.onkeydown = (ev: KeyboardEvent): any => {
+            GameApp.PressedSpace = (ev.key == " ");
+        }
+
+        Game.SetupGame();
+
+        // this is the ticker that runs once per frame, let's call our Update() function
+        this.app.ticker.add((delta) => {
+            Game.Update(delta);
+        });
+    }
+
+    static SetupGame()
+    {
+        // initial setup of the game state
+    }
+
+    static Update(delta: number)
```

Comment:
yeah, I wonder why you have this mix of static and instance variables and methods, is there anything am I missing?